src/dft/descriptor.cxx

```diff
@@ -218,6 +228,14 @@ void descriptor<prec, dom>::get_value(config_param param, ...) const {
             std::copy(values_.output_strides.begin(), values_.output_strides.end(),
                       va_arg(vl, std::int64_t*));
             break;
+        case config_param::FWD_STRIDES:
+            std::copy(values_.fwd_strides.begin(), values_.fwd_strides.end(),
+                      va_arg(vl, std::int64_t*));
+            break;
+        case config_param::BWD_STRIDES:
+            std::copy(values_.bwd_strides.begin(), values_.bwd_strides.end(),
+                      va_arg(vl, std::int64_t*));
+            break;
```

Comment:
I haven't updated this is the new PR because:
* We were non-spec-conformant in the same way with the I/O strides.
  * Users moving to FWD/BWD by just changing the config parameter name would have issues.
* For the `set_value` side, the spec is broken. I think its nicer if set/get value use the same parameter type.
* A much much better way of fixing this mess is to adopt your suggested updated type-safe spec for this.

Reply:
Agreed! That proposal will come in the hopefully near future.